README.md

```diff
@@ -1,10 +1,42 @@
-# jetcd - A Java client for etcd
+# jetcd - A Java Client for etcd
+[![Build Status](https://travis-ci.org/coreos/jetcd.svg?branch=master)](https://travis-ci.org/coreos/jetcd)
 
-jetcd requires JDK 8 to work correctly.
+jetcd is the official java client for [etcd](https://github.com/coreos/etcd)v3.
 
-[![Build Status](https://travis-ci.org/coreos/jetcd.svg?branch=master)](https://travis-ci.org/coreos/jetcd)
+> Note: jetcd is work-in-progress and may break backward compatibility.
+
+## Java Versions
+
+Java 8 or above is required.
+
+## Download
+
+### Maven
+```xml
+<dependency>
+  <groupId>com.coreos</groupId>
+  <artifactId>jetcd</artifactId>
+  <version>0.0.1</version>
+</dependency>
+```
+
+Development snapshots are available in [Sonatypes's snapshot repository](https://oss.sonatype.org/content/repositories/snapshots/).
+
+### Gradle
+
+TODO 
+
+### Manual
+
+Download latest jetcd jar from [releases](https://github.com/coreos/jetcd/releases) and all its dependent jars:
+
+[grpc-core-1.5.0.jar](http://repo1.maven.org/maven2/io/grpc/grpc-core/1.5.0/)
+[grpc-netty-1.5.0.jar](http://repo1.maven.org/maven2/io/grpc/grpc-netty/1.5.0/)
+[grpc-protobuf-1.5.0.jar](http://repo1.maven.org/maven2/io/grpc/grpc-protobuf/1.5.0/)
+[grpc-stub-1.5.0.jar](http://repo1.maven.org/maven2/io/grpc/grpc-stub/1.5.0/)
+[slf4j-api-1.7.2.jar](http://repo1.maven.org/maven2/org/apache/directory/studio/org.slf4j.api/1.7.2/)
+[slf4j-simple-1.7.25.jar](http://repo1.maven.org/maven2/org/slf4j/slf4j-simple/1.7.25/)
```

Comment:
do you mean just download jar from release is suffient? or `slf4j-simple-1.7.25.jar` is not needed?

```
### Manual

Download latest jetcd jar from [releases](https://github.com/coreos/jetcd/releases) 

```